install/terraform/open-match-build/open-match-build.tf

```diff
@@ -79,6 +96,34 @@ resource "google_service_account" "cluster_reaper" {
   #description = "Deletes orphaned GKE clusters."
 }
 
+# Create a Google service account with workload identity feature enabled to authenticate gcloud with its k8s service account binding.
+resource "google_service_account" "workload_identity_account" {
+  project      = "${var.gcp_project_id}"
+  account_id   = "${var.workload_identity_account_gsa}"
+  display_name = "${var.workload_identity_account_gsa}"
+}
+
+resource "google_service_account_iam_binding" "workload_identity_account_iam" {
+  service_account_id = "${google_service_account.workload_identity_account.name}"
+  role               = "roles/iam.workloadIdentityUser"
+
+  members = [
+    "serviceAccount:${var.gcp_project_id}.svc.id.goog[${var.workload_identity_account_namespace}/${var.workload_identity_account_ksa}]",
+  ]
+  depends_on = ["google_project_iam_binding.workload_identity_role_binding"]
+}
+
+# Grant the workload identity gsa iam.serviceAccounts.setIamPolicy permission for this project.
+resource "google_project_iam_binding" "workload_identity_role_binding" {
+  project = "${var.gcp_project_id}"
+  role    = "roles/iam.serviceAccountAdmin"
```

Comment:
I think the user that's applying the terraform template needs setIamPolicy permission but it's not required to setup a Kubernetes cluster since the binding is long term.

Reply:
Step 7 in https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity this guide requires setIamPolicy permission, the terraform setup that I changed is for the open match build so I think this should be added as a long term binding since we eventually may need the stress test in our test cycle.